run_server.py

```diff
@@ -2,9 +2,17 @@
 # uses wsgi to start up, and bypasses this file. So we are free to have debug
 # settings enabled.
 from Donut import app, config
+import argparse
+
+parser = argparse.ArgumentParser(description='Set up development server.')
+parser.add_argument('port_num', metavar='port_num', type=int, 
```

Comment:
Also you can configure argparse so that `python run_server.py -p 5000` and `python run_server.py --port 5000` do the same thing, following unix convention.